embed.fnc

```diff
@@ -707,6 +707,7 @@ ARdp	|SV **	|av_fetch	|NN AV *av				\
 CRdip	|SV **	|av_fetch_simple|NN AV *av				\
 				|SSize_t key				\
 				|I32 lval
+ARip	|SSize_t|AvFILL_	|NN AV *av
```

Comment:
Is this intended to be called directly?  If not should it be 'C' instead of 'A'?

Reply:
Thanks; corrected